storage/client/queries.go

```diff
@@ -291,6 +291,16 @@ func (qf QueryFactory) RuntimeBlocksQuery() string {
 		OFFSET $6::bigint`, qf.chainID, qf.runtime)
 }
 
+func (qf QueryFactory) RuntimeTransactionsQuery() string {
+	return fmt.Sprintf(`
+		SELECT round, tx_index, tx_hash, tx_eth_hash, raw, result_raw
+			FROM %s.%s_transactions
+			WHERE $1::bigint IS NULL OR round = $1::bigint
+		ORDER BY round DESC, tx_index DESC
```

Comment:
How do you feel about `ORDER BY round DESC, tx_index` instead to mirror the consensus tx endpoint?

Reply:
ok :shrug:

Reply:
no, now I feel worse about it: transactions within a block are executed in index order, so the logically latest transactions have higher index. the default operation of getting the "latest" queries would need tx_index DESC. 

also someone on the internet said that postgresql would need to create a separate index to go by round DESC, tx_index ASC https://stackoverflow.com/a/45598686/1864688 ):

Reply:
Makes sense. We can update the consensus tx endpoint to `tx_index DESC` as well then in a follow-up. I'll add it to the product agenda so we can confirm with them as well.